td-agent/manage-fluent-repositories.sh

```diff
@@ -122,13 +122,16 @@ EOF
 	find $FLUENT_RELEASE_DIR/4 -name "*$FLUENT_PACKAGE_VERSION*.rpm" | xargs rpm --resign --define "_gpg_name support@treasure-data.com"
 	find $FLUENT_RELEASE_DIR/4 -name "*$FLUENT_PACKAGE_VERSION*.rpm" | xargs rpm -K
 
-	# sign rpm repository
-	find $FLUENT_RELEASE_DIR/4 -name 'repomd.xml'
-	for f in `find $FLUENT_RELEASE_DIR/4 -name 'repomd.xml'`; do
-	    if [ -f "$f.asc" ]; then
-		rm -f $f.asc
+	# update & sign rpm repository
+	repodirs=`find "${FLUENT_RELEASE_DIR}" -regex "^${FLUENT_RELEASE_DIR}/4/\(redhat\|amazon\)/[2789]$"`
+	for repodir in $repodirs; do
+	    createrepo_c -v "${repodir}"
+
```

Comment:
In the previous versions, repodata/repomd.xml is generated under each x86_64 or aarch64.
Is it intentional change?, If so and it should be changed, it's ok for me.

Reply:
I mean: 

Before: td-agent-release/4/redhat/8/x86_64/repodata/repomd.xml
After:  td-agent-release/4/redhat/8/repodata/repomd.xml

Reply:
It's not intended. I'll fix it.

Reply:
Fixed in 63c1e084fe324c1c90a8c7686107710cddc8d672